scripts/run_formatter

```diff
@@ -4,6 +4,6 @@ version() {
   echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }';
 }
 
-if [ $(version $TRAVIS_ELIXIR_VERSION) -ge $(version "1.6.6") ]; then
+if [ $(version $TRAVIS_ELIXIR_VERSION) -ge $(version "1.8.0") ]; then
```

Comment:
I believe we don't run these files anymore right? Everything got moved to `scripts/test`

Reply:
That's right. This is a quick fix on this hotfix to fix a test failure on CI, so we don't have to base on develop right now. This code will be gone once we release a new minor version, as that'll have develop merged in.

Reply:
ah ok :D